benchmarks/gbench/CMakeLists.txt

```diff
@@ -23,8 +23,9 @@ if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
     add_custom_target(bench DEPENDS bench-results)
     add_custom_command(
       OUTPUT bench-results
-      COMMAND dr-bench analyze -n 1 -n 2 -n 3 -n 4 -s 1000000 -m mhp_gpu -m
-              mhp_cpu -m shp -f Stream_ -f Black_Scholes -f DotProduct_
+      COMMAND
+        dr-bench analyze -n 1 -n 2 -n 3 -n 4 -s 1000000 -m mhp_gpu -m mhp_cpu -m
+        shp -f Stream_ -f Black_Scholes -f DotProduct_ -f Inclusive_Scan
```

Comment:
icpx job failed because inclusive_scan does not work on shp (due to some unknown issue)

You can fix it by
- extract inclusive scan benchmarks from distributed_vector.cpp into separate file
- include this file in mhp benchmarks cmakelists.txt and not include it in shp cmakelistst.txt

Anyway it may be good to report issue (bug) about inclusive scan benchmarks in shp.